File: VendingMachineProject/Models/Transaction.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace VendingMachineProject.Models
{
    public partial class Transaction
    {
        public long TransactionId { get; set; }
        public long? MachineId { get; set; }
        public long? ItemId { get; set; }
        public decimal? AmountDeposited { get; set; }
        public decimal? AmountCharged { get; set; }
        public string TransactionStatus { get; set; }
        public DateTimeOffset? TransactionDate { get; set; }

        public virtual Item Item { get; set; }
        public virtual Machine Machine { get; set; }
    }
}

File: VendingMachineProject/ClientApp/src/app/models/Transaction.ts
export class Transaction {
  public transactionId: number;
  public machineId: number;
  public itemId: number;
  public totalAmount: number;
  constructor(transactionId: number, machineId: number, itemId: number, totalAmount: number){
    this.transactionId = transactionId;
    this.machineId = machineId;
    this.itemId = itemId;
    this.totalAmount = totalAmount;
  }
}
